tests/Unit/Framework/SetupLocalPythonEnvironment.cpp

```diff
@@ -47,6 +48,13 @@ SetupLocalPythonEnvironment::SetupLocalPythonEnvironment(
       pypp::from_py_object<std::vector<std::string>>(pyob_old_paths);
   std::string new_path =
       unit_test_src_path()+cur_dir_relative_to_unit_test_path;
+  if (not file_system::check_if_dir_exists(new_path)) {
+    ERROR("Trying to add path '" << new_path
```

Comment:
Make this `ERROR_NO_TRACE` for a less intimidating error message? :)

Reply:
Agreed. Done and rebased :)